lightstep/resource_metric_condition_test.go

```diff
@@ -717,7 +717,7 @@ resource "lightstep_metric_condition" "test" {
 				Config: updatedConditionConfig,
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckMetricConditionExists(resourceName, &condition),
-					resource.TestCheckResourceAttr(resourceName, "metric_query.0.tql", "spans count | rate 1h, 1h | filter (service == \"frontend\") | group_by [], sum | point (value + value) | reduce 30s, min"),
```

Comment:
This was failing before the query string comparison so it wasn't apparent that the expected query string wasn't right either!

Reply:
shouldnt we expect `value + value + value`? seems that's what the [formula update](https://github.com/lightstep/terraform-provider-lightstep/blob/03cc15b2d946c5c5de61952dfe50c8107d39dbcf/lightstep/resource_metric_condition_test.go#L685) is trying to do

Reply:
(i was only reading the left hand side... 😳 this is exactly what youre fixing)